packages/core/utils/src/common/define-config.ts

```diff
@@ -301,6 +301,21 @@ function resolveModules(
         ],
       },
     },
+    {
+      resolve: MODULE_PACKAGE_NAMES[Modules.CACHING],
+      options: {
+        providers: [
+          {
+            id: "caching-redis",
+            resolve: "@medusajs/medusa/caching-redis",
+            is_default: true,
+            options: {
+              redisUrl: process.env.REDIS_URL,
```

Comment:
```suggestion
              redisUrl: process.env.CACHE_REDIS_URL,
```

We might use different Redis clusters, so it's better to separate the two, even if they have the same value